airbyte-integrations/connectors/source-apple-search-ads/source_apple_search_ads/manifest.yaml

```
@@ -113,7 +113,7 @@ definitions:
         request_body_json:
           startTime: "{{ stream_slice.start_time }}"
           endTime: "{{ stream_slice.end_time }}"
-          timeZone: "UTC"
+          timeZone: "{{ config.time_zone }}"
```

```suggestion
          timeZone: "{{ config.get('time_zone', "UTC") }}"
```
Using this method it won't introduce a breaking change to existing connector.